numba/_helperlib.c

```diff
@@ -1518,6 +1518,159 @@ numba_xxgemm(char kind, char *transa, char *transb,
     return 0;
 }
 
+/*
+ * LAPACK calling helpers.  The helpers can be called without the GIL held.
+ * The caller is responsible for checking arguments (especially dimensions).
+ */
+
+/* Fetch the address of the given CLAPACK function, as exposed by
+   scipy.linalg.cython_lapack. */
+static void *
+import_clapack_function(const char *function_name)
```

Comment:
Since this is mostly duplicated from `import_cblas_function`, it would be nice to factor it out into a common function.